From f6ae67c66d263cf298fb479f7f698206b9916577 Mon Sep 17 00:00:00 2001
From: Joshua-Riek <jjriek@verizon.net>
Date: Wed, 11 Oct 2023 18:04:34 -0400
Subject: [PATCH 3/3] configs: rk3588: disable fdt and initrd relocation

---
 include/configs/rk3588_common.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/configs/rk3588_common.h b/include/configs/rk3588_common.h
index 46389d087d..74980fd604 100644
--- a/include/configs/rk3588_common.h
+++ b/include/configs/rk3588_common.h
@@ -24,7 +24,9 @@
 	"kernel_addr_r=0x02080000\0"	\
 	"ramdisk_addr_r=0x0a200000\0"	\
 	"kernel_comp_addr_r=0x08000000\0"	\
-	"kernel_comp_size=0x2000000\0"
+	"kernel_comp_size=0x2000000\0"	\
+	"fdt_high=0xffffffffffffffff\0"	\
+	"initrd_high=0xffffffffffffffff\0"
 
 #define CFG_EXTRA_ENV_SETTINGS \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
-- 
2.25.1

