From a2068631f1903a2fc256f32ab44bff89ea5018ce Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Fri, 23 Feb 2024 13:54:20 -1000
Subject: [PATCH 12/18] fix noble compile error

---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index 590fd4c1f2c..427cb75c6c1 100644
--- a/Makefile
+++ b/Makefile
@@ -389,6 +389,9 @@ export RCS_FIND_IGNORE := \( -name SCCS -o -name BitKeeper -o -name .svn -o    \
 			  -prune -o
 export RCS_TAR_IGNORE := --exclude SCCS --exclude BitKeeper --exclude .svn \
 			 --exclude CVS --exclude .pc --exclude .hg --exclude .git
+KBUILD_CFLAGS += $(call cc-option,-Wno-address)
+KBUILD_CFLAGS += $(call cc-option,-Wno-maybe-uninitialized)
+KBUILD_CFLAGS += $(call cc-option,-Wno-enum-int-mismatch)
 
 # ===========================================================================
 # Rules shared between *config targets and build targets
-- 
2.25.1

