From 5c29b088d647db40d29496b1516b7222f865a06f Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Sun, 23 Jul 2023 17:02:39 -0400
Subject: [PATCH] enable set local-mac-address

---
 common/fdt_support.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/common/fdt_support.c b/common/fdt_support.c
index e72aaf04..fa3d3e7a 100644
--- a/common/fdt_support.c
+++ b/common/fdt_support.c
@@ -705,8 +705,8 @@ void fdt_fixup_ethernet(void *fdt)
 
 			do_fixup_by_path(fdt, path, "mac-address",
 					 &mac_addr, 6, 0);
-			//do_fixup_by_path(fdt, path, "local-mac-address",
-			//		 &mac_addr, 6, 1);
+			do_fixup_by_path(fdt, path, "local-mac-address",
+					 &mac_addr, 6, 1);
 		}
 	}
 }
-- 
2.25.1

