From 522739f73dab148920b9a514770f3330bc0d355e Mon Sep 17 00:00:00 2001
From: Joshua Riek <jjriek@verizon.net>
Date: Thu, 18 May 2023 19:17:16 -0400
Subject: [PATCH 8/9] drivers: misc: fix khadas compile warning

---
 drivers/misc/khadas-mcu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/khadas-mcu.c b/drivers/misc/khadas-mcu.c
index 03f4eedbb62d..2ead9dca61e8 100644
--- a/drivers/misc/khadas-mcu.c
+++ b/drivers/misc/khadas-mcu.c
@@ -536,7 +536,7 @@ static void create_mcu_attrs(void) {
 		return;
 	}
 	for (i = 0; i < ARRAY_SIZE(mcu_class_attrs); i++) {
-		if (class_create_file(g_mcu_data->mcu_class, &mcu_class_attrs[i]));
+		if (class_create_file(g_mcu_data->mcu_class, &mcu_class_attrs[i]))
 			pr_err("create mcu attribute %s fail\n",
 							mcu_class_attrs[i].attr.name);
 	}
-- 
2.25.1

